python/pen_checker_cdc.py

```diff
@@ -152,10 +166,18 @@ def hmm_search_for_gene(fasta,gene):
         
         if hmm_match == "L":
             print(gff_base + '\tTrimethoprim resistant')
+            status = "R"
         elif hmm_match == "I":
             print(gff_base + '\tTrimethoprim sensitive')
+            status = "S"
         else:
             print(gff_base + '\tTrimethoprim unknown')
+            status = "NA"
+
+    print("Took this long for ORF finder: %s" % (toc_aa_creator - tic_aa_creator))
+    print("Took this long for HMM run: %s" % (toc_hmm_run - tic_hmm_run))
+
+    return status, gff_base
```

Comment:
Making sure the search returns the outcome of the HMMER match for resistance, along with the readable isolate id.